Makefile

```diff
@@ -388,12 +420,24 @@ MK_ALL_FLAMEC_OBJS        := $(MK_MAP_LAPACK2FLAMEC_OBJS) \
                              $(MK_ALL_FLAMEC_OBJS)
 endif
 
+ifeq ($(FLA_ENABLE_LAPACK2FLASH),yes)
+MK_ALL_FLAMEC_OBJS        := $(MK_MAP_LAPACK2FLASH_OBJS) \
+                             $(MK_MAP_LAPACK2FLASH_F2C_OBJS) \
+                             $(MK_MAP_LAPACK2FLASH_F2C_FLAMEC_OBJS) \
+                             $(MK_MAP_LAPACK2FLASH_F2C_INSTALL_OBJS) \
+                             $(MK_ALL_FLAMEC_OBJS)
+endif
+
 # BLAS
 ifeq ($(FLA_ENABLE_BUILTIN_BLAS),yes)
 ifeq ($(FLA_ENABLE_LAPACK2FLAME),no)
 MK_FLABLAS_F2C_OBJS       := $(MK_MAP_LAPACK2FLAMEC_F2C_INSTALL_OBJS) \
                              $(MK_FLABLAS_F2C_OBJS)
 endif
+ifeq ($(FLA_ENABLE_LAPACK2FLASH),no)
+MK_FLABLAS_F2C_OBJS       := $(MK_MAP_LAPACK2FLASH_F2C_INSTALL_OBJS) \
+                             $(MK_FLABLAS_F2C_OBJS)
+endif
```

Comment:
add a function that makes sure that if lapack2flash is enabled:
* lapack2flame is OFF
* SuperMatrix is ON
* external lapack is OFF

Reply:
Would this be better in the configure script? Or should I possibly do it in both?

Reply:
should be in the configure part

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/bd8b490c69720ca6210141fe13ac0703afc20b18

Reply:
Did not check for flame and flash both being enabled. Will fix.